knockout-pre-rendered.js

```diff
@@ -391,11 +471,19 @@
            value['field'] === undefined;
   }
 
-  function setExplicitObjectValues(value, viewModel) {
+  function setExplicitObjectValues(value, viewModel, allBindings) {
+    var propertyWriters = allBindings.get("_ko_prerender_initPropertyWriters");
```

Comment:
This also looks like a private field. See my previous question.

Reply:
Custom name internal to **knockout-pre-rendered**.